q

```diff
@@ -0,0 +1,230 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```

Comment:
You're learning, and at least you've got a sense of humour about it. I'm
certainly enjoying it :)
On Sat, 24 Oct 2015 at 06:17, Duke <notifications@github.com> wrote:

> In q
> <https://github.com/aexmachina/ember-notify/pull/66#discussion_r42903800>:
>
> > @@ -0,0 +1,230 @@
> > +
> > +                   S SU UM MM MA AR RY Y O OF F L LE ES SS S C CO OM MM MA AN ND DS S
>
> lol. no problems
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aexmachina/ember-notify/pull/66/files#r42903800>.
>